tests/suites/test_suite_psa_crypto_se_driver_hal.function

```diff
@@ -609,6 +613,20 @@ exit:
     return( ok );
 }
 
+/* Check that no persistent data exists for the given location. */
+static int check_no_persistent_data( psa_key_location_t location )
+{
+    psa_storage_uid_t uid = file_uid_for_location( location );
+    struct psa_storage_info_t info;
+    int ok = 0;
+
+    TEST_ASSERT( psa_its_get_info( uid, &info ) == PSA_ERROR_DOES_NOT_EXIST );
```

Comment:
This is ok, but in new code, we tend to prefer using `TEST_EQUAL` to compare the result of a function call with a constant, e.g. `TEST_EQUAL( …, PSA_ERROR_DOES_NOT_EXIST )`. This is partly stylistic and partly because when we get around to it we'll add the value of `…` to the output if the test fails.